common/src/main/java/dev/schmarrn/lighty/api/LightyColors.java

```diff
@@ -56,17 +56,23 @@ public static int getDangerARGB() {
     }
 
     public static int getARGB(int blockLightLevel, int skyLightLevel) {
-        int color = LightyColors.getSafeARGB();
+        // Artificial light is always safe
+        if (blockLightLevel > Config.getBlockThreshold()) return LightyColors.getSafeARGB();
+        // If we don't have artificial lighting, mobs can spawn at night
+        if (skyLightLevel > Config.getSkyThreshold()) return LightyColors.getWarningARGB();
+        // Without artificial nor skylight, mobs can always spawn
+        return LightyColors.getDangerARGB();
+    }
```

Comment:
Let me know if this is okay. The only reason I did it is to reduce indentation a bit. :)